examples/single/bundle/src/App.js

```diff
@@ -59,7 +59,9 @@ class App extends Component {
     } else if (this.state.session.isLoggedIn) {
       this.setState({ status: "dashboard", session });
     } else {
-      const authCode = new URL(window.location.href).searchParams.get("code");
+      const authCode =
+        new URL(window.location.href).searchParams.get("code") ||
```

Comment:
A comment here would be good (is it just that NSS and ESS use different param names for the same auth token value...?).

Reply:
I don't really like this. You have code handling authorization_code AND implicit flow as part of the same logical construct. It would be much better, design-wise to separate the different flows into discrete functions. Otherwise you end up conflating too many things right here

Reply:
This is more a temporary fix to be able to run tests than anything else. If things work the way I hope, we should be able to use the auth code flow on NSS as well, and just remove this because the demo would always use the auth code flow. Otherwise, i'll clean up the demo to be more structured.

Reply:
Acknowledged in e65ae11bb5e81297c1fbc3e19e0219f4d3b5282d